src/main/java/meditracker/command/AddCommand.java

```diff
@@ -41,8 +41,8 @@ public class AddCommand extends Command {
             new DosageEveningArgument(true),
             new ExpirationDateArgument(false),
             new IntakeFrequencyArgument(false),
-            new RepeatArgument(true),
-            new RemarksArgument(true)
+            new RemarksArgument(true),
+            new RepeatArgument(true)
```

Comment:
This was the reason why when I did the getRemarks, it shows the content of getRepeat instead. Good catch on this.

Reply:
This was not the issue causing the bug, but rather on [this line](https://github.com/AY2324S2-CS2113T-T09-1/tp/blob/3d31e0458ec0e554e3ef49eb8648a6a185218672/src/main/java/meditracker/command/AddCommand.java#L115) where the 2 arguments, remarks and repeat were flipped

Reply:
Got it. Thanks ya